.eslintignore

```diff
@@ -1,3 +1,4 @@
-assets/node_modules
-assets/elm-stuff
+client/node_modules
```

Comment:
Maybe use a single file for eslintignore and jshintignore ? This is recommended in https://eslint.org/docs/user-guide/configuring#using-an-alternate-file